src/PKSim.CLI.Core/Services/SimulationExporter.cs

```diff
@@ -137,15 +132,14 @@ public Task Export(Simulation simulation, string outputFolder, SimulationExportM
 
       private async Task exportSimulationResultsToCsv(Simulation simulation, string outputFolder)
       {
-         var simulationResultFile = CoreConstants.DefaultResultsExportNameFor(simulation.Name);
-         var simulationResultFileFullPath = csvPathUnder(outputFolder, simulationResultFile);
+         var simulationResultFileFullPath = csvPathUnder(outputFolder, simulation.Name);
```

Comment:
This change to make sure that the csv file name is the same as before (otherwise comparison using your fav tool won't be possible :)

Reply:
:man_dancing: